docs/getting-started/rbac/admin-role.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: getting-started
 rules:
 - apiGroups:
-  - tekton.dev
+  - triggers.tekton.dev
```

Comment:
I think we'll need to move the `- pipelineresources` here into a separate `apiGroup` for `tekton.dev`.

Reply:
moved